saleor/graphql/order/mutations/order_confirm.py

```diff
@@ -60,6 +62,34 @@ def get_instance(cls, info: ResolveInfo, **data):
             )
         return instance
 
+    @staticmethod
+    def charge_transaction_items(user, app, order, manager):
+        for transaction_item in order.payment_transactions.filter(
+            available_actions__contains=[TransactionAction.CHARGE],
+            authorized_value__gt=0,
+        ):
+            charge_value = transaction_item.authorized_value
+            if transaction_item.authorized_value > order.total.gross.amount:
+                charge_value = order.total.gross.amount
+            event = transaction_item.events.create(
```

Comment:
`saleor.graphql.payment.mutations.transaction.transaction_request_action.TransactionRequestAction` has `create_transaction_event_requested` method. It creates a new request event. Instead of doing the same thing here, we should extract this logic outside the mutation, and use it in both places. To keep the "create-event" logic in single place.